### PR TITLE
Restaurant MVP

### PR DESCRIPTION
**Pull request by cliftondavies**
### Completed:
- [x] Gitflow
- [x] Github Actions
- [x] ESLint
- [x] Stylelint
- [x] .gitignore
- [x] ES6 conventions
- [x] Modular JS app using webpack
- [x] README

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "file-loader": "^6.2.0",
+    "style-loader": "^2.0.0",
+    "webpack": "^5.4.0",
+    "webpack-cli": "^4.2.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
```
Comment (Dandush03) [inline package.json:27]: - [ ] **Flags:** Lets use flags for proper implementation of the code
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by Dandush03 — CHANGES_REQUESTED** [review]
# Status :red_circle: Require Changes :red_circle: 
### Great Job So Far! :muscle: 
**Check out my comments below and submit once you have fixed all the issues**
- [ ] **Flags:** You kind find [here](https://github.com/cliftondavies/Restaurant/pull/1#discussion_r520962309) the review
- [ ] **CSS Selectors:** Remember your best practices in the css section of this curse, avoid using CSS ID Selectors
# Happy Coding :nerd_face: :computer:

**Review by rOluochKe — APPROVED** [review]
Hello @cliftondavies 
### You got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Comment by cliftondavies** [comment]
### Thanks, @rOluochKe! 👍 🙏

**Pull request merged**